### PR TITLE
(maint) adds a script to update topics on repos

### PR DESCRIPTION
**Pull request by binford2k**
This uses the same repository inspection as labels.rb, so it accepts the
same options. It also adds another global option of `-g`/`--group` which
is a quick & simple switch between `modules.json` and `tools.json`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,13 @@ a single module repository in a few ways:
 * `--repo` (uses the first found of the `upstream` or `origin` remotes of CWD)
 * `--remote <name>` (uses the url of the remote name passed in the CWD)
 
+Topics
+-------
+
+Ensures a set of topics on repos. Works in a non destructive way unless you instruct
+it to delete topics. By default, if will run against IAC supported modules, but
```
Comment (mwaggett) [inline README.md]:
```suggestion
it to delete topics. By default, it will run against IAC supported modules, but
```

File: topics.rb [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+if options[:remote]
+  parsed = { options[:remote] => { 'github' => options[:remote] } }
+else
+  parsed = load_url(options)
+end
+
+util = OctokitUtils.new(options[:oauth])
+
+wanted_topics = [ 'hacktoberfest' ]
```
Comment (michaeltlombardi) [inline topics.rb:43]: Should this be configurable?

Comment (binford2k) [reply topics.rb:43]: yeah, it should, but so should the labels script and I wanted them to behave in the same way. That felt like a bigger change than fit into a single PR.

Comment (michaeltlombardi) [reply topics.rb:43]: Fair, I'm okay to merge this with the expectation that it will get those updates in the future.

**Pull request merged**